csharp/ql/test/library-tests/tostringwithtypes/toStringWithTypes.expected

```diff
@@ -22,10 +23,10 @@
 | M | M(uint) |
 | M | M(ulong) |
 | M | M(ushort) |
-| M | M<S>(params S[]) |
-| M | M<S>(ref S) |
-| M | M<char?>(ref char?) |
+| M<> | M<S>(params S[]) |
+| M<> | M<S>(ref S) |
+| M<Nullable<Char>> | M<char?>(ref char?) |
```

Comment:
First column is `toString`, second one is `toStringWithTypes`. Should we override `toString` on `SimpleType`s?
Should we override `toString` on `NullableType`?